### PR TITLE
Feat/tf 1565 tooling api update

### PR DESCRIPTION
**Pull request by Craig-J**

### CODE REVIEW

File: Examples-Web/BorderProgressButtons/src/index.tsx [hunk 1]
```diff
@@ -2,13 +2,13 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
-import TouchFree from "TouchFree/src/TouchFree";
+import { init } from "TouchFree/src/index";
```
Comment (cartmouse) [inline Examples-Web/BorderProgressButtons/src/index.tsx]: 🙈 This should be `... from "touchfree/src/index";`

File: Examples-Web/BorderProgressButtons/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "touchfree": "github:ultraleap/TouchFreeWebTooling"
+    "touchfree": "https://github.com/ultraleap/TouchFreeWebTooling/tarball/feat/TF-1028_api-audit"
```
Comment (cartmouse) [inline Examples-Web/BorderProgressButtons/package.json]: 🙈 Could point at `"https://github.com/ultraleap/TouchFreeWebTooling/tarball/develop"` instead

File: Examples-Web/WaveProgressButton/package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "touchfree": "github:ultraleap/TouchFreeWebTooling"
+    "touchfree": "https://github.com/ultraleap/TouchFreeWebTooling/tarball/feat/TF-1028_api-audit"
```
Comment (cartmouse) [inline Examples-Web/WaveProgressButton/package.json]: 🙈 Could point at `"https://github.com/ultraleap/TouchFreeWebTooling/tarball/develop"` instead

File: Examples-Web/IdleTimeout/package.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
-        "touchfree": "github:ultraleap/TouchFreeWebTooling"
+        "touchfree": "https://github.com/ultraleap/TouchFreeWebTooling/tarball/feat/TF-1028_api-audit"
```
Comment (cartmouse) [inline Examples-Web/IdleTimeout/package.json]: 🙈 Could point at `"https://github.com/ultraleap/TouchFreeWebTooling/tarball/develop"` instead

File: Examples-Web/BorderProgressButtons/src/index.tsx (hunk 1 above)
Comment (JordanBerni-UL) [reply Examples-Web/BorderProgressButtons/src/index.tsx]: Updated - don't need the trailing /index

**Review by cartmouse — APPROVED** [review]

**Pull request merged**